src/Controller/ItemController.php

```diff
@@ -2,7 +2,9 @@
 
 namespace App\Controller;
 
+use App\Model\Stercket;
 use App\Model\ItemManager;
+use App\Model\StercketManager;
```

Comment:
Why do you need Stercket in the Item controller ?
Do you even still need the item controller anymore ?